jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsLambdaHandlerResolver.kt

```diff
@@ -92,6 +99,15 @@ class NodeJsLambdaHandlerResolver : LambdaHandlerResolver {
         return true
     }
 
+    /**
+     * Whether the element is top level PSI element for a valid Lambda handler. It must be in the format as:
+     * export const lambdaHandler = functionExpression
+     */
+    private fun PsiElement.isValidTypeScriptLambdaHandler(): Boolean =
+        this is TypeScriptVariable &&
+            (this as? TypeScriptVariableImpl)?.isExported == true &&
+            this.lastChild.isLambdaFunctionExpression()
```

Comment:
It shows up as a `JSFunction`? even though it's typescript?

Reply:
`TypeScriptVariable` extends `JSVariable`